src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) {
+        String textFrom1 = "supply,30\n"
+                + "buy,10\n"
+                + "buy,13\n"
+                + "supply,17\n"
+                + "buy,10\n"
+                + "supply,5\n"
+                + "supply,80\n"
+                + "buy,39\n"
+                + "supply,56\n"
+                + "buy,32\n"
+                + "buy,11\n";
+        String textFrom2 = "buy,10\n"
+                + "supply,300\n"
+                + "buy,130\n"
+                + "buy,12\n"
+                + "supply,20\n"
+                + "buy,1\n"
+                + "supply,50\n"
+                + "supply,12\n"
+                + "supply,8\n"
+                + "buy,41\n"
+                + "supply,10\n"
+                + "buy,12\n"
+                + "buy,10\n"
+                + "buy,77\n"
+                + "supply,91";
+        File directory = new File("Directory1");
+        directory.mkdir();
```

Comment:
The creation of directories and files is a separate concern and should be placed in a separate method to follow the single responsibility principle.